.probo.yaml

```
@@ -54,6 +57,29 @@ steps:
       - chown www-data:www-data -R /var/www/html/sites/default/files/simpletest
       - source $ASSET_DIR/credentials.sh
```

This needs to be `test-credentials.sh`
```suggestion
      - source $ASSET_DIR/test-credentials.sh
```